kml/types_v8.hpp

```diff
@@ -21,7 +21,7 @@ struct BookmarkDataV8
                                   visitor(m_visible, "visible"),
                                   visitor(m_nearestToponym, "nearestToponym"),
                                   visitor(m_properties, "properties"),
-                                  visitor(m_compilations, "compilations"),
+                                  //visitor(m_compilations, "compilations"),
```

Comment:
So the HasCompilationsSection() should be enough, no?
Why comment here? Let's keep as-is.

Reply:
Not only compilations section is disabled but also Bookmark and Category fields list should be updated. If I enable back "compilations" field parsing would fail.